examples/nextjs-dashboard/pages/_app.tsx

```diff
@@ -0,0 +1,56 @@
+import type { AppProps } from "next/app";
+import Head from "next/head";
+import { RoomProvider } from "../src/liveblocks.config";
+import { useRouter } from "next/router";
+import React, { useMemo } from "react";
+import "./globals.css";
+import Example from "./index";
+
+function App({ Component, pageProps }: AppProps) {
+  const roomId = useOverrideRoomId("nextjs-dashboard");
+  return (
+    <RoomProvider
+      id={roomId}
+      initialPresence={{
+        selectedDataset: { cardId: null, dataKey: null },
```

Comment:
```suggestion
        selectedDataset: null,
```

We can make this `null` initially (see I changed the type in `liveblocks.config.ts`)